src/tests/time-base-api-test/time-base-api-test.c

```diff
@@ -48,6 +48,7 @@ static uint32 UT_TimerSync(osal_id_t timer_id)
      * calls the various functions on the first time through and stores the
```

Comment:
Update comment - UtAssert not fully thread-safe OBE and add reasoning for non-POSIX exclusion

Reply:
Fixed in commit 9a46055d